### DISCUSSION TITLE
Mapping multiple audio tracks into one

### DISCUSSION THREAD

**Discussion by KawaiianPizza** [RandomEngy/VidCoder#834] [Ideas]
I have videos with multiple audio tracks for different sources of files, desktop, music, discord. Could you add an easy way to map them together into a new audio stream i.e. discord + music on stream 0

**Comment by RandomEngy** [comment]
VidCoder uses the HandBrake engine, and muxing in new audio tracks isn't in scope for them. It's a transcoding application only.
